emscripten.py

```diff
@@ -448,6 +452,15 @@ def get_metadata(infile, outfile, modify_wasm, args):
   return metadata
 
 
+def create_sourcemapping_url_file(infile, url):
+  'Create a file with the contents of the sourceMappingURL section'
+  filename = os.path.join(shared.get_emscripten_temp_dir(), infile + '.smu')
+  with open(filename, 'wb') as f:
```

Comment:
How about `utils.write_binary(filename, leb128.u.encode(len(url) + url.encode('utf-8'))`.. maybe no better or worse?

Reply:
Changed to
```py
utils.write_binary(filename, leb128.u.encode(len(url)) + url.encode('utf-8'))
```
(I think you meant this)